.github/workflows/ci.yml

```diff
@@ -199,3 +199,26 @@ jobs:
         with:
           name: artifacts-capybara
           path: tmp/capybara
+  test_boot_without_db:
+    name: Boot web server without database
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: .node-version
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: .ruby-version
+          bundler-cache: true
+      - name: Cache js dependencies
+        uses: actions/cache@v4
+        with:
+          key: node_modules-${{ hashFiles('yarn.lock') }}
+          path: node_modules
+      - run: yarn install
+      - run: bin/test_boot
+        env:
+          HOST: http://www.rdv-test.fr
```

Comment:
Tout petit détail : on a toujours un retour à la ligne à la fin de nos fichiers, pour des [raisons diverses](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline), mais notamment pour éviter les interférences dans les diffs, ou ce genre de distraction :

![image](https://github.com/betagouv/rdv-service-public/assets/6357692/73c98bdb-1ec5-4d7d-aadc-72d4295bc3b4)